### PR TITLE
Allow dynamic text

### PR DESCRIPTION
**Pull request by RoiKed**
change hard coded text to localised text

### CODE REVIEW

File: KanCentralizedViewsPlugin/KanCastDialogViews/ChromecastDeviceDisconnectView.swift [hunk 1]
```diff
@@ -42,8 +42,8 @@ class ChromecastDeviceDisconnectView: UIView {
             return
         }
         let castDeviceName = currentSession.device.friendlyName ?? ""
-        
-        self.connectInformation?.text = "מחובר ל- \(castDeviceName)"
+        let connectToInfoText = ZAAppConnector.sharedInstance().localizationDelegate.localizationString(byKey: "device_msg_transmitter_to", defaultString: "מחובר ל - ")
+        self.connectInformation?.text = connectToInfoText! + castDeviceName
```
Comment (snooky23) [inline KanCentralizedViewsPlugin/KanCastDialogViews/ChromecastDeviceDisconnectView.swift]: Don't use `!`, please rewrite the code.

**Review by snooky23 — CHANGES_REQUESTED** [review]

File: KanCentralizedViewsPlugin/KanCastDialogViews/ChromecastDeviceDisconnectView.swift [hunk 2]
```diff
@@ -42,8 +42,8 @@ class ChromecastDeviceDisconnectView: UIView {
             return
         }
         let castDeviceName = currentSession.device.friendlyName ?? ""
-        
-        self.connectInformation?.text = "מחובר ל- \(castDeviceName)"
+        let connectToInfoText = ZAAppConnector.sharedInstance().localizationDelegate.localizationString(byKey: "device_msg_transmitter_to", defaultString: "מחובר ל - ")
```
Comment (snooky23) [inline KanCentralizedViewsPlugin/KanCastDialogViews/ChromecastDeviceDisconnectView.swift]: Const the key

File: KanCentralizedViewsPlugin/KanCastDialogViews/ChromecastDeviceDisconnectView.swift [hunk 3]
```diff
@@ -53,7 +53,8 @@ class ChromecastDeviceDisconnectView: UIView {
     }
     
     func disconnectButtonSetup() {
-        disconnectButton.titleLabel?.text = "התנתק"
+        let disconnectText = ZAAppConnector.sharedInstance().localizationDelegate.localizationString(byKey: "chromecast_actionSheet_disconnect_button", defaultString: "התנתק")
```
Comment (snooky23) [inline KanCentralizedViewsPlugin/KanCastDialogViews/ChromecastDeviceDisconnectView.swift]: Cont the key

File: KanCentralizedViewsPlugin/KanDialogView/KanDialogView.swift [hunk 4]
```diff
@@ -53,15 +53,15 @@ public class KanDialogView: UIView {
         if let configuration = configuration {
             if let title = configuration["sleep_mode_title"] as? String,
                 let titleLabel = titleLabel{
-                titleLabel.text = title
+                titleLabel.text = ZAAppConnector.sharedInstance().localizationDelegate.localizationString(byKey: "chromecast_actionSheet_title", defaultString: "המסך הזה קטן מדי בשבילי, נראה לי שאלך על")
```
Comment (snooky23) [inline KanCentralizedViewsPlugin/KanDialogView/KanDialogView.swift]: Cont the key

File: KanCentralizedViewsPlugin/KanDialogView/KanDialogView.swift [hunk 5]
```diff
@@ -53,15 +53,15 @@ public class KanDialogView: UIView {
...
                 titleLabel.setFont(fontNameKey: "action_text_sleep_mode_font_name",
                                    fontSizeKey: "action_text_sleep_mode_font_size",
                                    from: pluginStyles)
 
                 titleLabel.setColor(key: "action_text_sleep_mode_color",
                                     from: pluginStyles)
             }
-            if let closeButtonTitle = configuration["sleep_mode_close_title"] as? String,
+            if let closeButtonTitle = ZAAppConnector.sharedInstance().localizationDelegate.localizationString(byKey: "chromecast_actionSheet_cancel_button", defaultString: "סגירה"),
```
Comment (snooky23) [inline KanCentralizedViewsPlugin/KanDialogView/KanDialogView.swift]: Cont the key

**Review by snooky23 — CHANGES_REQUESTED** [review]

File: KanCentralizedViewsPlugin/KanCastDialogViews/ChromecastDeviceDisconnectView.swift (hunk 1 above)
Comment (RoiKed) [reply KanCentralizedViewsPlugin/KanCastDialogViews/ChromecastDeviceDisconnectView.swift]: done

File: KanCentralizedViewsPlugin/KanDialogView/KanDialogView.swift (hunk 5 above)
Comment (RoiKed) [reply KanCentralizedViewsPlugin/KanDialogView/KanDialogView.swift]: done

File: KanCentralizedViewsPlugin/KanDialogView/KanDialogView.swift (hunk 4 above)
Comment (RoiKed) [reply KanCentralizedViewsPlugin/KanDialogView/KanDialogView.swift]: done

**Review by snooky23 — APPROVED** [review]

**Pull request merged**